.vib/concourse/goss/web/goss.yaml

```diff
@@ -0,0 +1,26 @@
+file:
```

Comment:
I think you missed checking the `volumeMounts` both for the web and worker pods.

Reply:
As discussed, I will only add an entry for the `/bitnami/concourse/workdir` directory as the other one containing the keys it's being used by the application